kairon/shared/channels/broadcast/whatsapp.py

```
@@ -349,17 +385,17 @@ def __send_using_configuration(self, recipients: List):
             if self.config.get('collection_config'):
                 template_params, recipients = self.__prepare_template_params(raw_template, template_id)
             else:
-                template_params = self._get_template_parameters(template_config)
+                template_params = self.__prepare_template_params_with_template_config(raw_template, template_config)
 
-                template_params = template_params * len(recipients) if template_params \
-                    else [template_params] * len(recipients)
+                template_params = template_params if template_params else [template_params] * len(recipients)
 
             total = len(recipients)
             num_msg = len(list(zip(recipients, template_params)))
```

Ruff (0.14.3)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)